packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/__tests__/quanticCategoryFacet.test.js

```diff
@@ -213,11 +363,721 @@ describe('c-quantic-category-facet', () => {
         expect(facetContent).toBeNull();
       });
     });
+  });
+
+  describe('when the component is loading', () => {
+    beforeEach(() => {
+      searchStatusState = {
+        ...initialSearchStatusState,
+        isLoading: true,
+        hasError: false,
+        firstSearchExecuted: false,
+      };
+    });
+
+    it('should display the placeholder component', async () => {
+      const element = createTestComponent();
+      await flushPromises();
 
-    afterAll(() => {
-      functionsMocks.buildCategoryFacet.mockReturnValue(
-        categoryFacetControllerMock
+      const placeholder = element.shadowRoot.querySelector(
+        selectors.placeholder
       );
+      expect(placeholder).not.toBeNull();
+    });
+
+    describe('when the interface has an error', () => {
+      beforeEach(() => {
+        searchStatusState = {
+          ...initialSearchStatusState,
+          isLoading: true,
+          hasError: true,
+          firstSearchExecuted: false,
+        };
+      });
+
+      it('should not display the placeholder component', async () => {
+        const element = createTestComponent();
+        await flushPromises();
+
+        const placeholder = element.shadowRoot.querySelector(
+          selectors.placeholder
+        );
+        expect(placeholder).toBeNull();
+      });
+    });
+  });
+
+  describe('when the component is ready', () => {
+    describe('when the facet has values', () => {
+      beforeEach(() => {
+        facetState = {
+          ...initialFacetState,
+          valuesAsTrees: exampleFacetValues,
+        };
+      });
+
+      it('should display the facet card when the facet values have results', async () => {
```

Comment:
```suggestion
      it('should display the facet card', async () => {
```

is this part necessary if you already have the describe?